tests/omp/test_omp_tridiag.f90

```diff
@@ -216,21 +214,21 @@ program test_omp_tridiag
                    )
 
    call check_error_norm(du, sin_0_2pi, n, n_glob, n_block, 1, norm_du)
-   if (nrank == 0) print*, 'error norm hyperviscous', norm_du
+   if (nrank == 0) print *, 'error norm hyperviscous', norm_du
 
    if (nrank == 0) then
       if (norm_du > tol) then
          allpass = .false.
-         write(stderr, '(a)') 'Check 2nd ders, hyperviscous, dir-neu... failed'
+        write (stderr, '(a)') 'Check 2nd ders, hyperviscous, dir-neu... failed'
```

Comment:
seems like fprettify failed here for some reason, indentation level is wrong

Reply:
indeed, gonna fix that

Reply:
Fixed